src/doc/guide.md

```diff
@@ -104,41 +103,61 @@ Once you’re ready for release, you can use `cargo build --release` to compile
 <span style="font-weight: bold"
 class="s1">   Compiling</span> hello_world v0.1.0 (file:///path/to/project/hello_world)</code></pre>
 
+`cargo build --release` puts the resulting binary in
+`target/release/hello_world` instead of `target/debug`.
+
 # Working on an existing Cargo project
 
 If you download an existing project that uses Cargo, it’s really easy
 to get going.
 
-First, get the project from somewhere. In this example, we’ll use `color-rs`:
+First, get the project from somewhere. In this example, we’ll use `color-rs`
+cloned from its repository on GitHub:
```

Comment:
While we're doing this refresh, it might be a good idea to swap this example with a crate in the rust-lang org.